src/elm/State.elm

```diff
@@ -24,13 +24,16 @@ initModel =
     }
 
 
+port begin : () -> Cmd msg
```

Comment:
@sophielevens @maxgerber Can you make this name more declarative? Also could you make a `Ports.elm` and put all ports in there?